src/content/app/species/hooks/useSpeciesAnalytics.ts

```diff
@@ -78,14 +78,11 @@ const useSpeciesAnalytics = () => {
     });
   };
 
-  const trackSidebarToolstripButtonClick = (
-    iconName: string,
-    genomeId: string
-  ) => {
+  const trackSidebarModelOpen = (iconName: string) => {
     trackEvent({
-      category: 'species_page',
```

Comment:
Now that we have the correct app name set, the value for the `category` field doesn't have to be `species_page` for any of the other events. Could you update those as well?